src/textual/css/styles.py

```diff
@@ -384,6 +384,21 @@ def auto_dimensions(self) -> bool:
             has_rule("height") and self.height.is_auto  # type: ignore
         )
 
+    @property
+    def is_relative_width(self) -> bool:
+        """Does the node have a relative width?"""
+        width = self.width
+        return width is not None and width.unit in (
+            Unit.FRACTION,
+            Unit.PERCENT,
+        )
+
+    @property
+    def is_relative_height(self) -> bool:
+        """Does the node have a relative width?"""
+        height = self.height
+        return height is not None and height.unit in (Unit.FRACTION, Unit.PERCENT)
```

Comment:
Aren't `w`, `h`, `vh`, and `vw`, relative units too?

Reply:
`w` and `h` become percent units.

`vh` and `vw` aren't relative to the container, unless the container happens to be screen. Maybe the name could be clarified?

Reply:
A clarification is always welcome, in my opinion.